airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py

```diff
@@ -325,10 +325,17 @@ def transform_clickhouse(config: Dict[str, Any]):
             "port": config["port"],
             "schema": config["database"],
             "user": config["username"],
-            "secure": config["ssl"],
         }
         if "password" in config:
             dbt_config["password"] = config["password"]
+
+        # ssl is an optional configuration and is not present in strict-encrypt config
+        # if ssl option is not present in the config - default to True
+        if "ssl" in config:
+            dbt_config["secure"] = config["ssl"]
+        else:
+            dbt_config["secure"] = True
```

Comment:
Logic is fine here, one liner would be:
```python
dbt_config["secure"] = config.get("ssl", True)
```

Reply:
`+1` on the default value one-liner

Reply:
clearly, I don't code enough in Python :)